src/test/javascript/spec/component/programming-exercise/programming-exercise.component.spec.ts

```diff
@@ -238,15 +253,15 @@ describe('ProgrammingExercise Management Component', () => {
             comp.toggleProgrammingExercise(programmingExercise);
```

Comment:
Would you mind replacing the jasmine reference above with `toContainEntry(['id', programmingExercise.id)`?